vignettes/cookbook_preparatory_steps.Rmd

```diff
@@ -18,106 +18,108 @@ library(pacta.loanbook)
 
-The main input data sets required for the analysis are the following:
+The key required inputs include:
 
-### Asset-based Company Data (ABCD)
+### Asset-Based Company Data (ABCD)
 
-- required input
-- external source
+- Required input
+- External source
 - XLSX file
 
-This data set provides information on the production profiles and emission intensities of companies active in the following real economy sectors: Automotive (light-duty vehicles) manufacturing, aviation, cement production, coal mining, upstream oil & gas extraction, power generation, and steel production. The ABCD is typically obtained from third party data providers. However, it is possible to prepare the ABCD yourself or complement an external data set with entries that may not be covered out of the box.
+This dataset contains production profiles and emission intensities for companies in climate critical economic sectors: automotive (light-duty vehicles), aviation, cement, coal mining, upstream oil & gas, power generation, and steel. Typically sourced from third-party providers, it can also be self-prepared or supplemented with additional entries.
 
-The ABCD data set must be an XLSX file and contains the following columns:
+The ABCD dataset must be a `.xlsx` file that contains the following columns:
 
 ```{r cols_abcd, echo = FALSE, results = 'asis'}
 pacta.loanbook:::list_col_names_and_types(abcd_demo)
 ```
 
-For more detail about the necessary structure of this dataset, see the data dictionary for [abcd](data_abcd.html).
+PACTA is data-agnostic and supports any provider offering data in the correct format. One option is purchasing ABCD from [Asset Impact](https://asset-impact.gresb.com/).
 
-While PACTA is data agnostic and allows using data from any provider that offers the appropriate format, one option to obtain ABCD for this analysis is to buy the data from the data provider Asset Impact. Further information on how to obtain ABCD for PACTA and documentation of the individual sectors and data points can be found on the [Asset Impact website](https://asset-impact.gresb.com/).
+For details on the required structure of this dataset, refer to the [ABCD data dictionary](data_abcd.html).
 
 ### Scenario Data
 
-- required input(s)
-- external source
+- Required input(s)
+- External source
 - CSV file(s)
 
-The scenario data set provides information on the trajectories of technologies/fuel types and of emission intensity pathways for each of (or a subset of) the sectors covered in PACTA.
+For sectors with technology-level trajectories, the dataset provides Sectoral Market Share Percentage (SMSP) and Target Market Share Ratio (TMSR) pathways using the "market share approach". This allocation rule assumes that all companies in a sector must adjust production while maintaining constant market shares to align with the overall climate transition scenario.
```

Comment:
```suggestion
For sectors with technology-level trajectories, the dataset provides Sectoral Market Share Percentage (SMSP) and Target Market Share Ratio (TMSR) pathways using the "market share approach". This allocation rule assumes that all companies in a sector must adjust production while maintaining constant market shares within a sector to align with the overall climate transition scenario.
```
I think this should be spelled out a bit more clearly. Constant market share is a bit vague (sounds more like an overall financial term)